src/main/java/org/tron/common/runtime/Runtime.java

```diff
@@ -512,6 +513,7 @@ public void go() {
           result.getDeleteAccounts().clear();
           result.getLogInfoList().clear();
           result.resetFutureRefund();
+          result.rejectInternalTransactions();
 
           if (result.getException() != null) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move this left curly brace to the beginning of next line of code. ([squid:LeftCurlyBraceStartLineCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1633&issues=AWak_C2wdPPRVdwyhXMS&open=AWak_C2wdPPRVdwyhXMS)

[ISSUE_KEY]: AWak_C2wdPPRVdwyhXMS